packages/central-server/src/kobo/startSyncWithKoBo.js

```diff
@@ -170,6 +170,10 @@ export async function startSyncWithKoBo(models) {
     const koboDataServiceSyncGroups = await models.dataServiceSyncGroup.find({
       service_type: SERVICE_TYPE,
     });
+
+    // Put all kobo sync groups in an idle state, just in case they're stuck in an 'syncing' state due to a server shutdown
+    await Promise.all(koboDataServiceSyncGroups.map(async syncGroup => syncGroup.setSyncIdle()));
```

Comment:
It does feel a touch arbitrary/hacky doing this. However, this code is really designed to run once, upon server startup, which feels like the appropriate place to force the state to be `IDLE`.

Given that the state is just for application logic, perhaps there's a bit of a code smell here that we ought to manage the sync group state within memory rather than the DB? So that it wouldn't persist between server startups... But feels a bit much for right now